ytopt/benchmark/mmm-block-tl/notebook/tutorial-tl-Copy1.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
Why do we need this file in the repository?

Reply:
not needed. let's remove it.